components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -288,7 +288,8 @@ be lost if SCREAM_HACK_XML is not enabled.
     <cosp inherit="atm_proc_base">
       <cosp_subcolumns>10</cosp_subcolumns>
       <!-- Frequency at which to call COSP; positive values interpreted as number of steps, negative as number of hours -->
-      <cosp_frequency>-1</cosp_frequency>
+      <cosp_frequency>1</cosp_frequency>
+      <cosp_frequency_units>hours</cosp_frequency_units>
```

Comment:
Related to the comment below. I think you should take advantage of the XML attribute "valid_values", so that any invalid value will be caught during atmchange, rather than runtime. Here's an example of how to use it
```
 <atm_log_level valid_values="trace,debug,info,warn,error">info</atm_log_level>
```

Reply:
Good call. Done.